src/layers/src/geojson-layer/geojson-layer.ts

```diff
@@ -399,8 +455,26 @@ export default class GeoJsonLayer extends Layer {
     return this;
   }
 
+  isLayerHovered(objectInfo: any): boolean {
+    return this.isArrow
+      ? isLayerHoveredFromArrow(objectInfo, this.id)
+      : super.isLayerHovered(objectInfo);
```

Comment:
Does this mean that we could push the isArrow checks down into `super` and get support in all layers?

Reply:
Yes, I think we can do that, so we can support arrow format in other layers. We can do that later with new PRs.